spec/factories.rb

```diff
@@ -14,10 +16,18 @@
       declined true
     end
 
+    trait :requested do
+      requested true
+    end
+
     trait :pending do
       confirmed false
       declined false
     end
+
+    trait :unrequested do
+      requested false
+    end
   end
```

Comment:
Sort these traits? Would have the side-effect of putting `:requested` and `:unrequested` adjacent to one another.